grammar.js

```diff
@@ -321,6 +321,8 @@ module.exports = grammar({
       const hex_digits = /[a-fA-F0-9]+/;
       const hex_exponent_part = seq(choice('p', 'P'), signed_integer);
 
+      const bin_digits = /[01]+/;
```

Comment:
Does Lua have the concept of binary digits? Is there any documentation?

Reply:
I did not find any docs on it (yet), but here it seems to be valid: https://github.com/lua/lua/blob/e15f1f2bb7a38a3c94519294d031e48508d65006/testes/math.lua#L404

Reply:
Yeah, should probably just update `decimal_integer_literal`.